### PR TITLE
added the color brewer qualitative palettes

### LINKED ISSUE #4758: bokeh.palettes missing qualitative Brewer color maps
Is there a reason the qualitative color maps are not included in `bokeh.palettes`?  I can send a PR if they weren't deliberately omitted. One issue is they wouldn't fit the naming convention e.g. how do you name the `Dark2` with 5 colors? `Dark2_5`?

### PR DESCRIPTION
**Pull request by stvn66**
fixes #4758 
- [ added the qualitative palettes from ColorBrewer ] issues: 
- [ updated and modified the palette test (now checking the last int from the string name instead of the first int ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by canavandl** [comment]
Two line length errors (https://travis-ci.org/bokeh/bokeh/jobs/145278743#L1404). So close!

File: examples/charts/file/palettes.py [hunk 1]
```diff
@@ -26,7 +27,11 @@
                                  ("YlGnBu9", YlGnBu9), ("YlOrBr9", YlOrBr9),
                                  ("YlOrRd9", YlOrRd9), ("Inferno9", Inferno9),
                                  ("Magma9", Magma9), ("Plasma9", Plasma9),
-                                 ("Viridis9", Viridis9)])
+                                 ("Viridis9", Viridis9), ("Accent8", Accent8),
+                                 ("Dark8", Dark8), ("Paired9", Paired9),
+                                 ("Paired9", Paired9), ("Pastel1_9", Pastel1_9),
```
Comment (canavandl) [inline examples/charts/file/palettes.py]: Paired9 was added twice.

File: examples/charts/file/palettes.py [hunk 2]
```diff
@@ -26,7 +27,11 @@
                                  ("YlGnBu9", YlGnBu9), ("YlOrBr9", YlOrBr9),
                                  ("YlOrRd9", YlOrRd9), ("Inferno9", Inferno9),
                                  ("Magma9", Magma9), ("Plasma9", Plasma9),
-                                 ("Viridis9", Viridis9)])
+                                 ("Viridis9", Viridis9), ("Accent8", Accent8),
+                                 ("Dark8", Dark8), ("Paired9", Paired9),
```
Comment (canavandl) [inline examples/charts/file/palettes.py]: I think Dark is actually called Dark2 (though there doesn't seem to exist a Dark1): https://jiffyclub.github.io/palettable/colorbrewer/qualitative/#dark2_3

**Comment by canavandl** [comment]
Two tiny comments, otherwise LGTM (the failing integration tests aren't your fault, but because the SauceLab uploads don't work for non-Bokeh repo owners).

Thanks so much for your contribution. It was nice meeting you at the sprints today.

**Comment by caryan** [comment]
@stvn66 thanks for sending the PR.   The only other thing that came up in #4758 is that we're missing a license/attribution for the Brewer colours.  All I can find from Cynthia's  [webpage](http://www.personal.psu.edu/cab38/ColorBrewer/ColorBrewer_updates.html) is that it is Apache v2 licensed and we should add attribution.

I had this in my local copy:

```python
# This product includes color specifications and designs developed by Cynthia Brewer
# (http://colorbrewer2.org/).  The Brewer colormaps are licensed under the Apache v2 license.
# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 
```

**Comment by stvn66** [comment]
I just replaced Dark with Dark2, added the license for the Brewer colormaps to `palettes.py`, and removed the duplicate `Paired9` in the example file.

@canavandl, It was a pleasure to get to work on the project and become more familiar with the code base.

File: bokehjs/src/coffee/palettes/palettes.coffee [hunk 3]
```diff
@@ -454,8 +454,91 @@ palettes = {
...
+  Accent: {
+    Accent3     : [0x7fc97f, 0xbeaed4, 0xfdc086]
+    Accent4     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99]
+    Accent5     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0]
+    Accent6     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0, 0xf0027f]
+    Accent7     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0, 0xf0027f, 0xbf5b17]
+    Accent8     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0, 0xf0027f, 0xbf5b17, 0x666666]
+  }
+
+  Dark: {
```
Comment (canavandl) [inline bokehjs/src/coffee/palettes/palettes.coffee]: These also need to be renamed Dark2, if you have a sec. Otherwise it's ready to merge

Comment (bryevdv) [inline bokehjs/src/coffee/palettes/palettes.coffee]: friendly ping @stvn66 will you be able to make this last minor change? I would be nice to include this in `0.12.1` if possible, but that's in the next few days.

**Comment by canavandl** [comment]
LGTM. thanks again @stvn66

**Pull request merged**